appveyor.yml

```diff
@@ -0,0 +1,52 @@
+
+os: Visual Studio 2017
+
+environment:
+  matrix:
+    - ARCH:                x64
```

Comment:
* This does not seem to be used
* If you want to keep this, it seems like `platform` would be a more suitable than a custom environment variable

Reply:
I just copied it from dmd's appveyor.xml, I'll remove it.